boa3/compiler/codegenerator.py

```diff
@@ -576,15 +568,15 @@ def convert_store_variable(self, var_id: str):
         :param var_id: the value to be converted
         """
         is_arg = False
```

Comment:
I think we could create a function that returns this scope so we can reuse here and above

Reply:
In this case, the `scope` was only used to get the index, so I created a method that returns `index`, `local`, and `is_arg` instead.